voice/openai/src/index.ts

```diff
@@ -209,13 +185,14 @@ export class OpenAIVoice extends MastraVoice {
     const audioBuffer = Buffer.concat(chunks);
 
     const text = await this.traced(async () => {
-      const { filetype, ...otherOptions } = options || {};
-      const file = new File([audioBuffer], `audio.${filetype || 'mp3'}`);
+      const { response_format, ...otherOptions } = options || {};
```

Comment:
@Pom4H Can you maintain backwards compat here by also destructuring and conditionally using filetype

```
filetype || response_format || 'mp3'
```